test/ComponentTests/TestAssets/TestProjectForAssemblyResolution/UnitTest1.cs

```diff
@@ -5,7 +5,7 @@ namespace PlatformServices.Desktop.ComponentTests
 {
     [TestClass]
     [Serializable]
-    public class TestProjectForAssemblyResolution
+    public class TestProjectForAssemblyResolution : MarshalByRefObject
```

Comment:
Why we need this inheritance?

Reply:
This class is transferred across AppDomain boundaries in some test scenarios. It needs to be `[Serializable]` and inherit from `MarshalByRefObject`.